pvlib/iotools/solrad.py

```diff
@@ -49,8 +49,15 @@
 
 def read_solrad(filename):
     """
-    Read NOAA SOLRAD fixed-width file into pandas dataframe.  The SOLRAD
-    network is described in [1]_ and [2]_.
+    Read NOAA SOLRAD fixed-width file into pandas dataframe.
+
+    The SOLRAD network is described in [1]_ and [2]_.
+
+    .. versionchanged:: 0.10.4
```

Comment:
I like this PR, but it's breaking and should go into v0.11.0.

Should we forget v0.10.4 and make v0.11.0 the next release?

Reply:
I forgot 0.10.0 included several deprecations marked for removal in 0.11.0.  I agree it would be nice to wait a bit longer for that.

I also agree that this change is small, and in a function likely used by few.  But out of principle I'm reluctant to include any breaking change PR outside of a .0 release.  I suppose that means we put this PR on ice until we are ready to release 0.11.0.

Reply:
I guess I've mellowed with age and wouldn't delay this out of principle.  Would making the returned meta optional be an....option?